### PR TITLE
Official 1.19 Release Docs

### PR DESCRIPTION
**Pull request by savitharaghunathan**
🛑 DO NOT MERGE 🛑 UNTIL KUBERNETES 1.19 IS RELEASED
Target Date: 26-08-2020

Release deadlines
---
From [sig-release:](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19)

Date | Description
-- | --
JUN 19 | Open placeholder PRs
JUN 06 | PRs ready for review
JUL 16 | All PRs reviewed and ready to merge

More information
---
Contact @annajung @mikejoh @divya-mohan0209 @zestrells @jimangel @zacharysarah @kbarnard10 

/hold

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -156,12 +156,19 @@ js = [
...
+docsbranch = "release-1.19"
+url = "https://kubernetes.io"
+
 [[params.versions]]
 fullversion = "v1.18.0"
 version = "v1.18"
 githubbranch = "v1.18.0"
 docsbranch = "release-1.18"
-url = "https://kubernetes.io"
+url = "https://v-18.docs.kubernetes.io"
```
Comment (kbhawkey) [inline config.toml:171]: @savitharaghunathan , would you check this url?

Comment (savitharaghunathan) [reply config.toml:171]: Hi @kbhawkey, Thanks for catching it. I created PR to fix it - https://github.com/kubernetes/website/pull/23460 Can you take a look at it? I also fixed k8s minor versions along with it

Comment (savitharaghunathan) [reply config.toml:171]: Will wait for the fix to merge and then resolve this

**Comment by jimangel** [comment]
/lgtm
/approve

Per discussion [here](https://kubernetes.slack.com/archives/C1J0BPD2M/p1598466973088100?thread_ts=1598447979.070600&cid=C1J0BPD2M), I am manually merging this PR as part of the 1.19 release. 🎉

**Pull request merged**